ETC.tsv

```diff
@@ -14659,112 +14659,112 @@ ETC_20150918_014660	Fallen Legwyn Family Dungeon Notice Board
-ETC_20151001_014669	
-ETC_20151001_014670	
-ETC_20151001_014671	
-ETC_20151001_014672	
-ETC_20151001_014673	
-ETC_20151001_014674	
-ETC_20151001_014675	
-ETC_20151001_014676	
-ETC_20151001_014677	
-ETC_20151001_014678	
+ETC_20151001_014664	Lapemiter
+ETC_20151001_014665	Lapezard
+ETC_20151001_014666	Lapfighter
+ETC_20151001_014667	Lapflammer
+ETC_20151001_014668	Ferret Archer
+ETC_20151001_014669	Ferret Bearer
+ETC_20151001_014670	Ferret
+ETC_20151001_014671	Ferret Loader
+ETC_20151001_014672	Ferret Patter
+ETC_20151001_014673	Ferret Searcher
+ETC_20151001_014674	Ferret Slinger
+ETC_20151001_014675	Ferret Vendor
+ETC_20151001_014676	Lapeman
+ETC_20151001_014677	Member Alina Track
+ETC_20151001_014678	Chaser Werewolf
 ETC_20151001_014679	Kartas's Hunter Dullahan
-ETC_20151001_014680	
-ETC_20151001_014681	
-ETC_20151001_014682	
+ETC_20151001_014680	Ashaq Basement Prison 1F
+ETC_20151001_014681	Ashaq Basement Prison 2F
+ETC_20151001_014682	Ashaq Basement Prison 3F
 ETC_20151001_014683	Vine Forest Dungeon
 ETC_20151001_014684	Thorn Forest Dungeon
 ETC_20151001_014685	Zemyna, the Goddess of Earth..
-ETC_20151001_014686	
-ETC_20151001_014687	
-ETC_20151001_014688	
-ETC_20151001_014689	
-ETC_20151001_014690	
-ETC_20151001_014691	
-ETC_20151001_014692	
-ETC_20151001_014693	
-ETC_20151001_014694	
-ETC_20151001_014695	
-ETC_20151001_014696	
-ETC_20151001_014697	
-ETC_20151001_014698	
-ETC_20151001_014699	
-ETC_20151001_014700	
-ETC_20151001_014701	
-ETC_20151001_014702	
-ETC_20151001_014703	
-ETC_20151001_014704	
-ETC_20151001_014705	
-ETC_20151001_014706	
-ETC_20151001_014707	
-ETC_20151001_014708	
-ETC_20151001_014709	
-ETC_20151001_014710	
-ETC_20151001_014711	
-ETC_20151001_014712	
-ETC_20151001_014713	
-ETC_20151001_014714	
-ETC_20151001_014715	
-ETC_20151001_014716	
-ETC_20151001_014717	
-ETC_20151001_014718	
-ETC_20151001_014719	
-ETC_20151001_014720	
-ETC_20151001_014721	
-ETC_20151001_014722	
-ETC_20151001_014723	
-ETC_20151001_014724	
-ETC_20151001_014725	
-ETC_20151001_014726	
-ETC_20151001_014727	
+ETC_20151001_014686	Did you really think you could take me!
+ETC_20151001_014687	Goddess, please give me strength..
+ETC_20151001_014688	Karail's soul has been tortured.
+ETC_20151001_014689	Destroy the Holy Relics of the Church Order
+ETC_20151001_014690	Defeat the appeared Margiris!
+ETC_20151001_014691	They are coming!
+ETC_20151001_014692	Margiris!
+ETC_20151001_014693	You lured the enemies to our position..{nl}You useless fool!
+ETC_20151001_014694	Surrondings have been quiet after used a slient scroll!{nl}Please feel free and defeat Gagoil!
+ETC_20151001_014695	Escape from here!!
+ETC_20151001_014696	Stone Frost is gaining a weight.{nl}Move to safe place!
+ETC_20151001_014697	Monsters which sense somebody's presence are coming!
+ETC_20151001_014698	A weak Stone Frost poured from the box!{nl}Defeat monsters to avoid Stone Frost!
+ETC_20151001_014699	A lot of monster are coming due to the sound of destroyed intensive spirit's device!
```

Comment:
A large amount of monsters are coming due to the sound of the destroyed intensive spirit's device!